iceoryx_posh/test/moduletests/test_popo_subscriber_port.cpp

```diff
@@ -87,20 +100,29 @@ TEST_F(SubscriberPortSingleProducer_test, initialStateNoChunksLost)
     EXPECT_FALSE(m_sutUserSideSingleProducer.hasLostChunksSinceLastCall());
 }
 
-TEST_F(SubscriberPortSingleProducer_test, initialStateReturnsNoCaProMessage)
+TEST_F(SubscriberPortSingleProducer_test, initialStateReturnsNoCaProMessageWhenNoSubOnCreate)
```

Comment:
The test names should be written in UpperCamelCase

Reply:
And this one will be dedicated to @FerdinandSpitzschnueffler and to Ronny because of his birthday